framework/requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+-r requirements.txt
+
+freezegun==0.3.15
+py==1.11.0
+pykwalify==1.7.0
+pytest==5.4.3
+pytest-aiohttp==0.3.0
+pytest-asyncio==0.15.1
+pytest-html==2.1.1
+pytest-trio==0.7.0
+tavern==1.2.2
```

Comment:
Tavern package installation needs to be reviewed since launching framework unit test with it cause the following error.

```
================================================== short test summary info ==================================================
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_actions-params0-expected_result0] - tavern.util....
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_actions-params1-expected_result1] - tavern.util....
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_actions-params2-expected_result2] - tavern.util....
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_actions-params3-expected_result3] - tavern.util....
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_resources-params4-expected_result4] - tavern.uti...
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_resources-params5-expected_result5] - tavern.uti...
FAILED framework/wazuh/tests/test_security.py::test_rbac_catalog[get_rbac_resources-params6-expected_result6] - tavern.uti...
================================== 7 failed, 2085 passed, 55 warnings in 250.56s (0:04:10) ==================================
```

And all errors have the same body:

```
>       raise exceptions.BadSchemaError(error)
E       tavern.util.exceptions.BadSchemaError: Error at /home/eduardoleon/git/wazuh/api/api/spec/spec.yaml:11623 - column 28 - cannot define an empty value in test - either give it a value or explicitly set it to None
```

Reply:
The API schema had one field with no value assigned, adding 'None' to it solved the issue.